saleor/graphql/payment/tests/benchmark/test_payment_transactions.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from ....tests.utils import get_graphql_content
+
+NUMBER_OF_ORDERS = 11
```

Comment:
It's not good idea to add the global variable for one test, also this is still magic number.